app/components/ErrorLayout/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './style.scss';
+
+const ErrorLayout = ({ error }) => {
+  if (!error) return null;
```

Comment:
No need to check this. ErrorLayout will only be rendered if some error occurred.

Reply:
Done